documentation/asciidoc/topics/ref_logging.adoc

```diff
@@ -0,0 +1,52 @@
+[id='logging-levels_{context}']
+= Logging reference
+Find information about log categories and levels.
+
+.Log categories
+[%header,cols=3*]
+|===
+|Root category
+|Description
+|Default level
+
+|`org.infinispan`
+|{brandname} messages
+|`INFO`
+
+|`org.jgroups`
+|Cluster transport messages
+|`INFO`
+
+|===
+
+.Log levels
+[%header,cols=2*]
+|===
+|Log level
+|Description
+
+|`TRACE`
```

Comment:
Values for log level inside Infinispan CR should be lower case.

Reply:
nice catch. thanks @dmvolod